### PR TITLE
chore: make refresh notebook still commit successful notebooks on fail

### PR DESCRIPTION
**Pull request by jfrery**
closes https://github.com/zama-ai/concrete-ml-internal/issues/3764

### CODE REVIEW

**Comment by jfrery** [comment]
I am refreshing the notebooks here with the new workflow and will cherry-pick the commit to another PR.

Also need to revert the on: push.

File: script/make_utils/jupyter.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ WHAT_TO_DO="open"
 
 # Create a list of notebooks with long execution times in order not to consider them when refreshing
 # all notebooks at the same time.
-LONG_EXECUTION_TIMES_NOTEBOOKS=()
+LONG_EXECUTION_TIMES_NOTEBOOKS=("ClassifierComparison.ipynb")
```
Comment (RomanBredehoft) [inline script/make_utils/jupyter.sh:18]: do we need to do something here ? should we add a fixme then ?

Comment (jfrery) [reply script/make_utils/jupyter.sh:18]: Let me remove this now that knn is gone from the comparison notebook.

**Review by RomanBredehoft — CHANGES_REQUESTED** [review]
I'm not sure these changes makes successful notebooks be committed even if another fails ? or at least I can't see what changes would make this possible here !

**Comment by jfrery** [comment]
> I'm not sure these changes makes successful notebooks be committed even if another fails ? or at least I can't see what changes would make this possible here !

Yes they have been committed in #304 while Deployment notebook failed.

File: script/make_utils/jupyter.sh [hunk 2]
```diff
@@ -75,7 +80,11 @@ then
         echo "Refreshing ${NOTEBOOK}"
 
         START=$(date +%s)
-        jupyter nbconvert --to notebook --inplace --execute "${NOTEBOOK}"
+        if jupyter nbconvert --to notebook --inplace --execute "${NOTEBOOK}"; then
+            echo "${NOTEBOOK}" >> "${SUCCESSFUL_NOTEBOOKS}"
+        else
+            echo "${NOTEBOOK}" >> "${FAILED_NOTEBOOKS}"
+        fi
         END=$(date +%s)
```
Comment (jfrery) [inline script/make_utils/jupyter.sh:88]: This is what makes the refresh notebook not crash on failure @RomanBredehoft .

**Review by RomanBredehoft — APPROVED** [review]
I find it weird that this makes it possible but I trust you ! thanks a lot for this !!

**Pull request merged**